.github/workflows/master.yml

```diff
@@ -47,16 +47,25 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
           registry-url: https://registry.npmjs.org/ # Needed for auth
+
       - name: yarn install
         run: yarn install --frozen-lockfile
-      - run: yarn lint
-      - run: yarn build
-      - run: yarn test
+
+      - name: lint
+        run: yarn lerna run lint
```

Comment:
not clear on why this isn't just `yarn lint`

Reply:
`yarn lint` as actually supposed to be different, hadn't pushed the latest update :grin: